rm/rm-server/build.gradle

```diff
@@ -21,8 +21,8 @@ dependencies {
     testCompile "org.objectweb.proactive:programming-extension-pnpssl:${programmingVersion}"
     testCompile project(':common:common-api').sourceSets.test.output
 
-    testCompile 'org.jruby:jruby-complete:9.0.5.0'
-    testCompile 'org.python:jython-standalone:2.7.0'
+    testCompile 'org.jruby:jruby-complete:9.1.12.0'
+    testCompile 'org.python:jython-standalone:2.7.1'
```

Comment:
Here it seems correct, don't use jython, but jython-standalone:
https://mvnrepository.com/artifact/org.python/jython-standalone